libs/serialization/include_compatibility/hpx/runtime/serialization/detail/preprocess_container.hpp

```diff
@@ -0,0 +1,23 @@
+//  Copyright (c) 2019 Ste||ar Group
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#pragma once
+
+#include <hpx/config.hpp>
+#include <hpx/serialization/config/defines.hpp>
+#include <hpx/serialization/detail/preprocess_container.hpp>
+
+#if HPX_SERIALIZATION_HAVE_DEPRECATION_WARNINGS
+#if defined(HPX_MSVC)
+#pragma message(                                                               \
+    "The header hpx/runtime/serialization/detail/preprocess_container.hpp is deprecated, \
+    please include hpx/serialization/detail/preprocess_container.hpp instead")
+#else
+#warning                                                                       \
+    "The header hpx/runtime/serialization/detail/preprocess_container.hpp is deprecated, \
+    please include hpx/serialization/detail/preprocess_container.hpp instead"
+#endif
+#endif
```

Comment:
We can skip this one if you like. I did remove all the detail compatibility headers in #4762.

Reply:
I have removed all of compatibility headers in serialization/detail now.